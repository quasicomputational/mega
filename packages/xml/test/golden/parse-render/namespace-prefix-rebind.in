<foo xmlns:a="example:x"><a:bar xmlns:a="example:b" /></foo>