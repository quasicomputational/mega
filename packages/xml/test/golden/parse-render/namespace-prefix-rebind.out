<foo><n1:bar xmlns:n1="example:b"></n1:bar></foo>